src/pages/Waiting/index.js

```diff
@@ -1,16 +1,21 @@
 import React, { useState, useEffect } from "react";
 import styled from "styled-components";
-import { useHistory } from "react-router-dom";
+import { useHistory, useParams, useLocation } from "react-router-dom";
 import socket from "../../components/utils/socket";
 import Team from "../../components/Team";
+import Header from "../../components/Header";
 
 export default function WaitingRoom() {
   const [isReady, setIsReady] = useState(false);
   const [team, setTeam] = useState(0);
   const [canStart, setCanStart] = useState(false);
 
-  const channelId = 1;
-  const roomId = 1;
+  const location = useLocation();
+  const { id } = useParams();
+  const channelRoom = location.state.channel;
+
+  const channelId = id.split("-")[0];
+  const roomId = id.split("-")[1];
```

Comment:
```suggestion
  const [channelId, roomId ] = id.split("-");
```
아마 이런식으로도 쓸 수 있을거예요~!

Reply:
아! 참고해보겠습니다!! 감사합니다.